src/models/Scholarship.ts

```diff
@@ -0,0 +1,21 @@
+interface ScholarshipProps {
+  // TODO(https://github.com/beyondhb1079/s4us/issues/56):
+  // Update this to reflect the schema
+  id?: string;
+  name: string;
+  description: string;
+  deadline: string;
+  website: string;
+  school: string;
+  year: string;
+}
+
+export default class Scholarship {
+  id?: string;
```

Comment:
How come we need another id when the ScholarshipProps object already has one?

Reply:
What I initially had in this PR was a bit offset of what I intended. What I mean to have here is an id property separate from the the construction of a scholarship (since the ID is generated, not provided). The latest version of this PR reflects that.